detox/android/detox/proguard-rules-app.pro

```diff
@@ -1,4 +1,5 @@
 -keepattributes InnerClasses, Exceptions
+
```

Comment:
You shouldn't change this file.

Reply:
Thanks Yarik. I don't remember changing this one so I apologize for that.